source/agora/consensus/pool/Transaction.d

```diff
@@ -125,7 +135,8 @@ public class TransactionPool
             this.updateSpenderList(tx);
             auto tx_hash = tx.hashFull();
             iota(tx.outputs.length).each!(idx => utxo_set[UTXO.getHash(tx_hash, idx)] = tx.outputs[idx]);
-            this.known_txs[tx_hash] = Set!Hash.from(tx.inputs.map!(input => input.utxo));
+            Amount fee_rate = Amount(this.db.execute("SELECT fee FROM tx_pool WHERE key = ?", tx_hash).oneValue!ulong);
```

Comment:
Noting this down for future startup/shutdown improvements; these kind of IO bound loops in ctors seems problematic for node startup procedure. But no effects for the current condition